channeldb/invoices.go

```diff
@@ -9,6 +9,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/lightningnetwork/lnd/channeldb/entities"
```

Comment:
Something like `models` could also work here perhaps, would also be a shorter import.

Reply:
changed